core-bundle/src/Image/ImageFactory.php

```
@@ -232,6 +242,10 @@ private function createConfig($size, ImageInterface $image): array
      */
     private function createImportantPart(ImageInterface $image): ?ImportantPart
     {
+        if (!$this->framework->isInitialized()) {
+            return null;
```

why not boot the framework here?

---

I don’t think that would be a good idea. Do you think it is necessary?

---

I don't think it is.

---

Well either the framework is always bootet at this point already, or we need to boot it. If you really don't want to, then we should throw an exception and not just behave as if the image does not exist in the database imho.

---

> Well either the framework is always bootet at this point already, or we need to boot it.

Hm, I think @aschempp has a point here.

---

I added an exception in cc478877a2a0fe754286211d97f0dbfdff8e416e
And it now only queries the database if the image is inside the upload folder in which case booting the framework is not neccesary.